src/Http/Controllers/V1/Shop/Customer/CartController.php

```diff
@@ -25,6 +24,16 @@ public function __construct(
     ) {
     }
 
+    /**
+     * Resource class name.
+     *
+     * @return string
+     */
+    public function resource()
+    {
+        return 'Webkul\RestApi\Http\Resources\V1\Shop\Checkout\CartResource';
```

Comment:
Please imort namespace and use class here,
CartResource::class

Reply:
Done